okio/src/commonTest/kotlin/okio/FakeFileSystemTest.kt

```diff
@@ -488,6 +488,21 @@ abstract class FakeFileSystemTest internal constructor(
     }
   }
 
+  @Test
+  fun closesWithUseBlock() {
+    fun testMethodWithUse() {
+      val sink = fakeFileSystem.sink(base / "all-files-includes-file")
```

Comment:
Does this test have anything to do with FakeFileSystem? Seems like we should have a test for `use`.

Reply:
Yep, you got me. :)